lighthouse-core/test/results/sample_v2.json

```diff
@@ -3287,7 +3300,7 @@
         }
       ],
       "id": "best-practices",
-      "score": 0
+      "score": null
```

Comment:
Why is this null?

Reply:
I believe this file is generated by `update:sample-json` so, not something I specifically did in this PR. Perhaps I am wrong? @paulirish @brendankenny

Reply:
That's right @schalkneethling, but it's null because your audit is erroring when run on our fixtures 
 because you added a new artifact. Not ideal that we're adding an erroring audit to our golden, do you think you could manually add an expected Doctype artifact to https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/test/results/artifacts/artifacts.json?

Reply:
Thank you for the feedback @patrickhulce. I am happy to add it but, I probably need some guidance. Thanks!